### PR TITLE
User Guide Skeleton

### PR DESCRIPTION
**Pull request by lipwei1808**
Added `add_transaction` UG feature as well

### CODE REVIEW

File: docs/images/unicash/command-outputs/addTransactionFailedOutput1.png
Comment (sp4ce-cowboy) [inline docs/images/unicash/command-outputs/addTransactionFailedOutput1.png:1]: If you're on macOS, there's a way to take really nice `.png` shortcuts natively. macOS will add shadows and space the screenshot appropriately, and only the window been screenshot needs to be clicked, no dragging or anything required. The keyboard shortcut should be as below. It either copies to clipboard or saves the image to Desktop.

image


Even a boring JDK API window looks so clean and evenly spaced. 

image

 
Just a minor aesthetic suggestion, if we could all use this form of taking screenshots, I think we can achieve a really consistent and clean look for all images in our UG/DG.

**Review by sp4ce-cowboy — APPROVED** [review]
LGTM. Just a minor aesthetic consideration.

File: docs/images/unicash/command-outputs/addTransactionFailedOutput1.png
Comment (lipwei1808) [reply docs/images/unicash/command-outputs/addTransactionFailedOutput1.png:1]: Cool suggestion

File: docs/images/unicash/command-outputs/addTransactionSuccessOutput2.png
Comment (sp4ce-cowboy) [inline docs/images/unicash/command-outputs/addTransactionSuccessOutput2.png:1]: These look great!

**Review by sp4ce-cowboy — APPROVED** [review]
LGTM! Screenshots looking good

**Pull request merged**